.eslintrc.json

```diff
@@ -10,7 +10,9 @@
         "no-plusplus": 0,
         "vars-on-top": 0,
         "no-unused-vars": 0,
-        "max-lines-per-function": 0
+        "max-lines-per-function": 0,
+        "import/no-cycle": ["error", { "maxDepth":2}]
```

Comment:
Why do we need this?

Reply:
@tachyons, could you please check this?

Reply:
Hi @syntax-bot 
the dependency cycle itself exist on main codebase, But the  imported generateImage function may not be impacted by that.

The cycle is like -
```
[Line 7 saveimage.js] -> [Line 2 save.js] -> [Line 18 setup.js] ->  [Line 2 save.js]  !! this one
```

I think you can just ignore that instead of disabling eslint rule, do not need to modify the configuration, as that can lead to ignore other genuine issues of codebase.